code/datums/actions/items/vortex_recall.dm

```diff
@@ -5,11 +5,11 @@
 	button_icon_state = "vortex_recall"
 
 /datum/action/item_action/vortex_recall/IsAvailable(feedback = FALSE)
-	var/area/current_area = get_area(target)
-	if(!current_area || current_area.area_flags & NOTELEPORT)
+	if(!istype(target, /obj/item/hierophant_club))
+		return
```

Comment:
if the only way for this to be on a different item is admin interference; I say let it work.